srcs/python/kungfu/tensorflow/optimizers/async_sgd.py

```diff
@@ -5,12 +5,13 @@
                                       request_variable_with_template,
                                       save_variable)
 
-from .core import KungFuOptimizer, defuse, fuse
+from .core import KungFuOptimizer, defuse, fuse, _tf_assign
 
 
 def get_random_peer(cluster_size, self_rank):
-    t = tf.random_uniform([], minval=0, maxval=cluster_size, dtype=tf.int32)
-    return tf.cond(tf.equal(t, self_rank), lambda: tf.mod(t + 1, cluster_size),
+    t = tf.random.uniform([], minval=0, maxval=cluster_size, dtype=tf.int32)
```

Comment:
I change tf.random_uniform to tf.random.uniform.

I also change tf.mod to tf.math.floormod.

Could you check if they are equivalent? The old APIs are not available in TF2. @lgarithm